conda/recipes/pynvjitlink/meta.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2018-2023, NVIDIA CORPORATION.
+
+{% set py_version = environ['CONDA_PY'] %}
+{% set cuda_version = '.'.join(environ['RAPIDS_CUDA_VERSION'].split('.')[:2]) %}
+{% set cuda_major = cuda_version.split('.')[0] %}
```

Comment:
This is always 12 right? If so, think we can drop this and just set it in `conda_build_config.yaml` to use everywhere else

```suggestion
```